packages/amplication-client/src/Resource/EntitiesTile.tsx

```diff
@@ -23,6 +23,7 @@ const EVENT_DATA: TrackEvent = {
 
 function EntitiesTile({ resourceId }: Props) {
   const history = useHistory();
+  const {workspace,project} = useParams();
```

Comment:
is that how we are fetching the parameters now? 
aren't we using the context? 
If this stays, I think we should use a type

Reply:
i will change it to use context @yuval-hazaz

Reply:
@yuval-hazaz  Done